packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/Link/Anchor.tsx

```diff
@@ -30,8 +30,8 @@ const Anchor = ({
   return (
     <a
       className="d-flex"
-      target="_blank"
-      rel="noreferrer"
+      target={link.external ? '_blank' : '_self'}
+      rel={link.external ? 'noopener' : 'opener'}
```

Comment:
I think that `opener` rel has no effect with a `_self` target (https://developer.mozilla.org/fr/docs/Web/HTML/Attributes/rel#opener).
In the case of external link, `rel` attribute should be empty